lib/routes/routeBackbeat.js

```diff
@@ -128,6 +140,38 @@ function _checkMultipleBackendRequest(request, log) {
     return undefined;
 }
 
+function handleTaggingOperation(request, response, method, dataStoreVersionId,
+    log, callback) {
+    const storageLocation = request.headers['x-scal-storage-class'];
+    const objectMD = {
+        dataStoreName: storageLocation,
+        location: [{ dataStoreVersionId }],
+    };
+    if (method === 'Put') {
```

Comment:
just a small comment, `method` may confuse a bit because the HTTP method for a put object tagging is actually `POST`. we could keep the operation semantics (`puttagging` or `deletetagging`) instead

Reply:
I agree and I've renamed it to `type`. I was following the naming scheme for the `objectTagging` method in the multiple backend gateway.